lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -57,10 +59,10 @@ def serializable_hash_for_collection(serializer, options)
           hash
         end
 
-        def serializable_hash_for_single_resource(serializer, options)
+        def serializable_hash_for_single_resource(options)
           primary_data = primary_data_for(serializer, options)
           relationships = relationships_for(serializer)
-          included = included_for(serializer)
+          included = included_for
```

Comment:
Not required for this PR, but passing the serializer reference along is useless here as it is available via an `attr_reader`.
Here, `included_for` could be renamed to `included`.